java/src/Ice/src/main/java/com/zeroc/IceInternal/Instance.java

```diff
@@ -1880,4 +1894,16 @@ private NetworkProxy createNetworkProxy(com.zeroc.Ice.Properties properties, int
     private static boolean _oneOffDone = false;
     private QueueExecutorService _queueExecutorService;
     private QueueExecutor _queueExecutor;
+
+    private Map<String,String[]> _builtInModulePackages = java.util.Collections.unmodifiableMap(new HashMap<String, String[]>() {{
+        put("Glacier2", new String[] { "com.zeroc" });
+        put("Ice", new String[] { "com.zeroc" });
+        put("IceBox", new String[] { "com.zeroc" });
+        put("IceDiscovery", new String[] { "com.zeroc" });
+        put("IceGrid", new String[] { "com.zeroc" });
+        put("IceLocatorDiscovery", new String[] { "com.zeroc" });
+        put("IceMX", new String[] { "com.zeroc" });
```

Comment:
In #1963 we can updates this to `put("IceMX", new String[] { "com.zeroc.Ice", "com.zeroc.Glacier2", "com.zeroc.IceStorm" });`